packages/frontend/editor-ui/src/components/CanvasChat/future/components/LogsPanelActions.vue

```diff
@@ -20,33 +26,67 @@ const popOutButtonText = computed(() => locales.baseText('runData.panel.actions.
 const toggleButtonText = computed(() =>
 	locales.baseText(isOpen ? 'runData.panel.actions.collapse' : 'runData.panel.actions.open'),
 );
+const menuItems = computed(() => [
+	{
+		id: 'toggleSyncSelection' as const,
+		label: locales.baseText('runData.panel.actions.sync'),
+		checked: isSyncEnabled,
+	},
+	...(showPopOutButton ? [{ id: 'popOut' as const, label: popOutButtonText.value }] : []),
+]);
+
+function handleSelectMenuItem(selected: string) {
+	// This switch looks redundant, but needed to pass type checker
```

Comment:
You could use generic props for ActionDropdown to type `selected`, but I see in the commits you tried, so probably it didn't work?